lib/game_console.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require_relative '../dependencies'
+
+class GameConsole
+  include Validation
+  include Database
+  include GameStart
+
+  def initialize(name, difficulty)
+    @game = Game.new(name: name, difficulty: difficulty)
+  end
+
+  def start
+    loop do
+      break if @game.attempts.zero? || @game.win
+
+      start_info(@game.attempts, @game.hints)
+      input = gets.chomp
+      case input
+      when GameMenu::EXIT then break close
+      when 'hint' then next hint_info(@game.use_hint)
+      when /^[1-6]{4}/ then check_info(@game.check(input))
+      else next puts I18n.t(:wrong_process) unless guess_is_valid?(input)
+      end
+    end
+    puts I18n.t(:game_over)
+    statistics
+  end
+
+  def check_info(check)
+    puts check
+  end
+
+  def hint_info(use_hint)
+    puts use_hint
+  end
+
+  def start_info(attempts, hints)
+    puts I18n.t(:game_process, attempts: attempts, hints: hints)
+  end
+
+  def statistics
+    summary_info(@game.secret)
+    if @game.win
+      puts I18n.t(:win)
+      print I18n.t(:save)
+      save_results if gets.chomp == 'save'
+    else
+      puts I18n.t(:lose)
+    end
+  end
+
+  def summary_info(secret)
+    puts I18n.t(:secret, secret: secret)
+  end
+
+  def save_results
+    attempts_total = calc_counts(@game.difficulty)[:attempts]
```

Comment:
you don't need this method, just use the DIFFICULTY_LEVEL constant
Also you can try to do it in one line using `.values_at`

Reply:
i didn't find how to do this with values_at. i tried several examples, but it's given other results than i want.

Reply:
i did with the DIFFICULTY_LEVEL constant